### ISSUE TITLE
Table 1 results training setting in the paper

### ISSUE THREAD

**Issue by jameslahm** [HCPLab-SYSU/HCP-MLR-PL#4]
Thank you for your great work!
I don't find the training setting for the results in Table 1 in the paper Semantic-Aware Representation Blending for Multi-Label Image Recognition with Partial Labels. Would you mind sharing the training setting? Is it under 40% proportions of the dropped labels? Thanks!
![image](https://user-images.githubusercontent.com/43805318/182082748-4b43e5f5-6e14-4d1e-a26b-5d1071546780.png)
